src/fast-path.js

```diff
@@ -397,15 +400,17 @@ FPp.needsParens = function(assumeExpressionContext) {
 
     default:
       if (
-        parent.type === "NewExpression" && name === "callee" &&
+        parent.type === "NewExpression" &&
+          name === "callee" &&
           parent.callee === node
       ) {
         return containsCallExpression(node);
       }
   }
 
   if (
-    assumeExpressionContext !== true && !this.canBeFirstInStatement() &&
+    assumeExpressionContext !== true &&
+      !this.canBeFirstInStatement() &&
```

Comment:
It has to. Otherwise you get stuff like this:

```js
var x = foo &&
bar && 
baz;
```

The rule is that it always indents the expressions after the first one. This has a neat way of showing precedence rules too:

```js
foo ||
bar &&
  baz &&
  hizzle ||
mizzle
```

In the above it clarifies that the `bar && baz && hizzle` expression will be evaluated first before the other expressions with `||`. I really like how this turned out, and complex math expressions look really nice because of it. It's one of those things where we can convey semantic meaning to the user.
```